content/docs/components/hyperparameter-tuning/experiment.md

```diff
@@ -0,0 +1,477 @@
+  or minimized based on `type`) and the corresponding hyperparameter set
+  in `Experiment.status`. If the `objectiveMetricName` metric for a set of
+  hyperparameters exceeds the `goal`, Katib stops trying more hyperparameter 
+  combinations. See the [EarlyStoppingSetting 
+  type](https://github.com/kubeflow/katib/blob/master/pkg/apis/controller/common/v1alpha3/common_types.go#L40)
+  and the [ObjectiveSpec 
+  type](https://github.com/kubeflow/katib/blob/master/pkg/apis/controller/common/v1alpha3/common_types.go#L47).
+
+* **algorithm**: The search algorithm that you want Katib to use to find the
+  best hyperparameters or neural architecture configuration. Examples include
+  random search, grid search, Bayesian optimization, and more.
+  See the [search algorithm details](#search-algorithms) below.
+  
+* **trialTemplate**: The template that defines the trial.
+  You must package your ML training code into a Docker image, as described 
+  [above](#docker-image). You must configure the model's
+  hyperparameters either as command-line arguments or as environment variables,
+  so that Katib can automatically set the values in each trial.
+
+    You can use one of the following job types to train your model:
+
+    * [Kubernetes Job](https://kubernetes.io/docs/concepts/workloads/controllers/jobs-run-to-completion/)
+      (does not support distributed execution). 
+    * [Kubeflow TFJob](/docs/guides/components/tftraining/) (supports 
+      distributed execution).
+    * [Kubeflow PyTorchJob](/docs/guides/components/pytorch/) (supports 
+      distributed execution).
+    
+    See the [TrialTemplate 
+    type](https://github.com/kubeflow/katib/blob/master/pkg/apis/controller/experiments/v1alpha3/experiment_types.go#L165-L179).
+    The template 
+    uses the [Go template format](https://golang.org/pkg/text/template/).
+    
+    You can define the job in raw string format or you can use a 
+    [ConfigMap](https://kubernetes.io/docs/tasks/configure-pod-container/configure-pod-configmap/).
+
+* **parallelTrialCount**: The maximum number of hyperparameter sets that Katib
+  should train in parallel.
+
+* **maxTrialCount**: The maximum number of trials to run.
+  This is equivalent to the number of hyperparameter sets that Kabit should
+  generate to test the model.
+
+* **maxFailedTrialCount**: The maximum number of failed trials before Katib
+  should stop the experiment.
+  This is equivalent to the number of failed hyperparameter sets that Kabit 
+  should test.
+  If the number of failed trials exceeds `maxFailedTrialCount`, Katib stops the
+  experiment with a status of `Failed`.
+
+* **metricsCollectorSpec**: A specification of how to collect the metrics from
+  each trial, such as the accuracy and loss metrics.
+  See the [details of the metrics collector](#metrics-collector) below.
+
+* **nasConfig**: The configuration for a neural architecture search (NAS).
```

Comment:
Should clarify here that NAS is an alpha feature.

Reply:
Done.